package.json

```diff
@@ -41,9 +41,15 @@
     "LeanKit"
   ],
   "scripts": {
-    "build": "gulp",
-    "test": "gulp test",
-    "coverage": "gulp coverage"
+    "prebuild": "npm run lint",
+    "build": "webpack --config webpack.config.js",
+    "pretest": "npm run lint",
+    "test": "karma start --no-auto-watch --single-run || true",
```

Comment:
I'd like to understand more about why the `|| true` was needed? I wasn't seeing error output from npm when it was removed.